VideoWeb/VideoWeb.EventHub/Models/CallbackEvent.cs

```diff
@@ -16,5 +19,10 @@ public class CallbackEvent
         public string TransferTo { get; set; }
 
         public string Reason { get; set; }
+        public bool IsParticipantInVmr { get; set; }
+        public ConferenceState ConferenceStatus { get; set; }
+        public IEnumerable<Participant> OtherParticipantsInVmr { get; set; }
```

Comment:
I don't think we need this as it's only used to set up IsOtherParticipantsInConsultationRoom

Reply:
Hi Farron, IsParticipantInVmr  and ConferenceStatus  are using in JoinedEventhandler, I will remove OtherParticipantsInVmr

Reply:
Yeah I actually just meant that one, I must have marked the others by mistake.